lib/local/platform/windows.js

```diff
@@ -1,6 +1,14 @@
 var os = require('os');
+var fs = require('fs');
 var path = require('path');
 var programFiles = os.arch() === "x64" ? process.env["ProgramFiles(x86)"] : process.env.ProgramFiles;
+var windowsDirectory = process.env.winDir;
+var systemApps = path.join(windowsDirectory, 'SystemApps');
+var microsoftEdgeDirectory = fs.readdirSync(systemApps).filter(function(folder) {
+    return folder.startsWith('Microsoft.MicrosoftEdge');
+  }).map(function(folder) {
+    return path.join(systemApps, folder);
+  })[0];
```

Comment:
This finds the `MicrosoftEdge.exe` but you can't run that as it's a Windows Store application. It seems I need to provide an executable for the launchpad code to think that the browser exists. This makes sense as the executable won't exist on previous versions of Windows.